src/libsync/creds/keychainchunk.h

```diff
@@ -73,13 +73,29 @@ class Job : public QObject
     }
 #endif
 
+    /**
+     * @return Whether this job autodeletes itself once finished() has been emitted. Default is true.
+     * @see setAutoDelete()
+     */
+    bool autoDelete() const {
```

Comment:
Misplaced opening brace, also probably a good idea to put the implementation in the cpp file anyway.